### ISSUE TITLE
Cannot use an already registered cmap that have a dot '.' in its name

### ISSUE THREAD

**Issue by stefraynaud** [lukelbd/proplot#142]
https://github.com/lukelbd/proplot/blob/3f065778edce6e07b109e436b504238c7194f04f/proplot/styletools.py#L2212

It is necessary to check if a given cmap name is is that of an already registered
colormap before trying to load it from disk.
All cmocean colormaps are registered with a dot in their name, and therefore wen cannot load them through proplot wrappers.

````python
import cmocean
import numpy as np
import proplot as pplt
fig, axs = pplt.subplots()
axs.contourf(np.arange(5*4).reshape(4,5), cmap='cmo.thermal')
````

**Comment by lukelbd** [comment]
I remember adding this code and thinking "No one uses dots in their colormap names, right? Right. It will be fine..."

I'd still like to keep the loading-from-file feature, but this conflict can be repaired for 99% of cases by replacing that line with:

```python
if '.' in arg and os.path.isfile(arg):
```

Also, you should note that proplot already [loads the cmocean colormaps](https://proplot.readthedocs.io/en/latest/colormaps.html#Included-colormaps) into matplotlib's colormap database, so you can just use `cmap='thermal'`.

**Comment by lukelbd** [comment]
This is fixed by 972956b1, specifically these lines:

https://github.com/lukelbd/proplot/blob/972956b1a35ba1fb5323db33bb70fcfabb48cce7/proplot/colors.py#L1858-L1868
